src/vmspawn/vmspawn.c

```diff
@@ -697,6 +700,51 @@ static int setup_notify_parent(sd_event *event, int fd, int *exit_status, sd_eve
         return 0;
 }
 
+static int bus_open_in_machine(sd_bus **ret, unsigned cid, unsigned port, const char *private_key_path) {
+        _cleanup_(sd_bus_flush_close_unrefp) sd_bus *bus = NULL;
+        _cleanup_free_ char *ssh_escaped = NULL, *bus_address = NULL;
+        char port_str[DECIMAL_STR_MAX(unsigned)], cid_str[DECIMAL_STR_MAX(unsigned)];
+        int r;
+
+        r = sd_bus_new(&bus);
+        if (r < 0)
+                return r;
+
+        const char *ssh = secure_getenv("SYSTEMD_SSH") ?: "ssh";
+        ssh_escaped = bus_address_escape(ssh);
+        if (!ssh_escaped)
+                return -ENOMEM;
+
+        xsprintf(port_str, "%u", port);
+        xsprintf(cid_str, "%u", cid);
+
+        bus_address = strjoin(
+                "unixexec:path=", ssh_escaped,
+                ",argv1=-xT",
```

Comment:
Having a comment here explaining what these flags do would help since they don't see to have longform equivalents.

Reply:
I have added a comment explaining what they do now. For reference -x disables X11 forwarding and -T disables TTY allocation.